.gitignore

```diff
@@ -1,6 +1,7 @@
 *.pyc
 build
 dist
+tags
```

Comment:
that's an autogenerated ctags file, I used it at some point, but actually I can just revert this piece, doesn't matter